README.md

```diff
@@ -1 +1,245 @@
-README
\ No newline at end of file
+# Flask-Restless-Zx #

```

Comment:
Looks like the bulk of this could have remained in the original readme file rather than shifting it all over?

Reply:
Fewer changes from the original make code reviews quicker and makes it easier for this to potentially become a standard part of the original package we forked from.